### PR TITLE
Remove dependabot config during merge

### PR DESCRIPTION
**Pull request by arajkumar**
Dependabot is a GH app which automatically monitors repos with `.github/dependabot.yaml` files and keeps raising pull requests incase if it finds outdated dependencies or dependencies with vulnerabilities. However Openshift has a PST based workflow for security notification and remediation. 

Fixes https://github.com/rhobs/syncbot/issues/23

Ref https://github.com/openshift/prometheus/pull/115

Signed-off-by: Arunprasad Rajkumar <arajkuma@redhat.com>

### CODE REVIEW

File: .github/workflows/merge-flow.yaml [hunk 1]
```diff
@@ -94,6 +94,11 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: ${{ inputs.go-version }}
+      - name: Remove dependabot configuration
+        continue-on-error: true
+        run: |
+          rm -fr .github/dependabot.*
```
Comment (slashpai) [inline .github/workflows/merge-flow.yaml:100]: Even though there is no difference in command I think `rm -rf` is better to read :)

Comment (sthaha) [reply .github/workflows/merge-flow.yaml:100]: do you need a `-r` ?

**Comment by sthaha** [comment]
@arajkumar could you please elaborate why this is need? 

> Openshift org doesn't rely on dependabot for version bumps. 

doesn't really explain what issues it cause and how removing these files fixes it

File: .github/workflows/merge-flow.yaml (hunk 1 above)
Comment (arajkumar) [reply .github/workflows/merge-flow.yaml]: `-r` is not required. I've fixed it. Thanks @sthaha and @slashpai !

File: .github/workflows/merge-flow.yaml [hunk 2]
```diff
@@ -94,6 +94,11 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: ${{ inputs.go-version }}
+      - name: Remove dependabot configuration
+        continue-on-error: true
+        run: |
+          rm -r .github/dependabot.*
```
Comment (sthaha) [inline .github/workflows/merge-flow.yaml:100]: did you mean delete `-r` and keep `-f` ?

Comment (arajkumar) [reply .github/workflows/merge-flow.yaml]: I suppose use force, but since I have `continue-on-error: true`, I can remove `-f` too.

**Review by sthaha — APPROVED** [review]

**Pull request merged**